code/modules/food_and_drinks/drinks/drinks.dm

```diff
@@ -321,6 +338,16 @@
 		crushed_can.icon_state = icon_state
 		qdel(src)
 	..()
+	
+
+/obj/item/weapon/reagent_containers/food/drinks/soda_cans/attack_self(mob/user)
+	if(!is_open_container())
+		to_chat(user, "You pull back the tab of \the [src] with a satisfying pop.")
+		flags |= OPENCONTAINER
+		playsound(src, pick(open_sounds), 50, 1)
```

Comment:
Use playsound(src, "can_open", ... and create a "can_open" entry in get_sfx: https://github.com/tgstation/tgstation/blob/master/code/game/sound.dm#L101